### PR TITLE
Update tls-1.0-and-1.1-deprecation-for-office-365.md

### PR DESCRIPTION
**Pull request by WorkShay**
Specifying the staged implementation of the rollout.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ appliesto:
 
 # TLS 1.0 and 1.1 deprecation for Office 365
 > [!IMPORTANT]
-> We temporarily halted deprecation enforcement of TLS 1.0 and 1.1 for commercial customers due to covid-19, but as supply chains have adjusted and certain countries open back up, we are resetting the TLS enforcement to start Oct 15, 2020. 
+> We temporarily halted deprecation enforcement of TLS 1.0 and 1.1 for commercial customers due to covid-19, but as supply chains have adjusted and certain countries open back up, we are resetting the TLS enforcement to begin Oct 15, 2020 and rollout will continue over the weeks and months.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/tls-1.0-and-1.1-deprecation-for-office-365.md]:
```suggestion
> We temporarily halted deprecation enforcement of TLS 1.0 and 1.1 for commercial customers due to covid-19, but as supply chains have adjusted and certain countries open back up, we are resetting the TLS enforcement to begin Oct 15, 2020 and rollout will continue over the following weeks and months. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@WorkShay Please merge this Pr. Thanks.

**Comment by WorkShay** [comment]
Hi Yogesh,

I don’t have the permissions to do this. @KC<mailto:KC.Cross@microsoft.com> or someone else does.

Shayna

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

**Pull request merged**